src/Console/Controller.php

```diff
@@ -91,6 +108,9 @@ public function actionListen($cwd = null, $timeout = null, $env = [])
         while (true) {
             $this->stdout("Running new process...\n");
             $this->runQueueFetching($command, $cwd, $timeout, $env);
+            if ($this->sleepTimeout) {
```

Comment:
How about we set the default value to `0` instead of `NULL`, and then check if timeout greater than 0?